chall03/srouhe.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"net/http"
+	"io/ioutil"
+)
+
+
+func errCheck(err error) {
+	if err != nil {
+        log.Fatalf("Fatal error: %v", err)
+    }
+}
+
+func parseResp(respBody string) (string) {
+
+	re, err := regexp.Compile("id=(\\d*).*r=(\\d{1,3}).*g=(\\d{1,3}).*b=(\\d{1,3})")
```

Comment:
so far, the best regex I've seen; but if you plan to do something like that you should maybe use `fmt.Sscanf()` instead of regex.

I suggest reading: https://softwareengineering.stackexchange.com/a/113243

Reply:
Thanks for helpful feedback! Sscanf() would've been smoother indeed and good points in the article. Hopefully there's plenty of go in upcoming challenges :)